examples/volume_slice4.py

```diff
@@ -40,6 +40,7 @@ def mousePressEvent(self, event):  # noqa: N802
 
 camera = gfx.OrthographicCamera(128, 128)
 camera.position.set(64, 64, 128)
+camera.scale.y = -1  # in this case we tweak the camera, not the plane
```

Comment:
does that not mess up mouse interaction though?

Reply:
Thinking of CT scans, for example, you may want to flip axis to get the correct view. I think the camera scale is where this should happen. Maybe it Just Works with interaction? Otherwise I suppose interaction should take it into account, ideally.

Reply:
Well, let's merge this as is and we can refer to #101 later